src/iOS/Xamarin.iOS/Samples/Analysis/DistanceMeasurement/DistanceMeasurement.cs

```diff
@@ -48,6 +48,8 @@ public DistanceMeasurement()
 
         private void Initialize()
         {
+            _mySceneView.GeoViewTapped -= MySceneView_GeoViewTapped;
```

Comment:
Initialize is consistently called after `ViewWillAppear`, so I don't think this is necessary. I would recommend removing this, and also removing the event subscription at the end of `Initialize`. I recommend adding a comment in place of the event subscription at the end of `Initialize` indicating that the event subscription happens in `ViewWillAppear`. 

I also recommend replacing the measurement changed handler lambda with a method and subscribing/unsubscribing in `ViewWillAppear`/`ViewDidDisappear`.